otel/attribute_uint64.go

```diff
@@ -0,0 +1,18 @@
+package otelattr
```

Comment:
Looks like it's not a `chore` anymore.

Also, better to keep this pkg under `internal`.

Reply:
Agree with `internal`

Reply:
dn